src/content/docs/apis/nerdgraph/examples/nerdgraph-streaming-export.mdx

```diff
@@ -155,6 +216,27 @@ mutation {
 }
 ```
 
+Azure Event Hub:
+```graphql
+mutation {
+  streamingExportUpdateRule(
+    id: YOUR_NEW_RELIC_ACCOUNT_ID,
```

Comment:
`id` is a rule id, not an account id. In the examples for other mutations that take a rule id that is referred to as `INSERT_RULE_ID`. Looks like it was first mistakenly added to the AWS export update example above so it should be changed there as well.